<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.revature.service.ClientAccountServiceTest" tests="6" skipped="0" failures="0" errors="0" timestamp="2022-03-17T11:14:52" hostname="LAPTOP-6318QO8H" time="2.63">
  <properties/>
  <testcase name="test_getAccountByClientIdAccountId_InvalidId()" classname="com.revature.service.ClientAccountServiceTest" time="2.562"/>
  <testcase name="test_deleteAccountOfClient_sqlException()" classname="com.revature.service.ClientAccountServiceTest" time="0.022"/>
  <testcase name="test_getAccountByClientId_DoesNotExist()" classname="com.revature.service.ClientAccountServiceTest" time="0.002"/>
  <testcase name="test_getAccountByClientIdAccountId_AccountDoesNotExist()" classname="com.revature.service.ClientAccountServiceTest" time="0.003"/>
  <testcase name="test_getAccountByyClientIdAccountId_sqlException()" classname="com.revature.service.ClientAccountServiceTest" time="0.003"/>
  <testcase name="testgetAccountByClientId()" classname="com.revature.service.ClientAccountServiceTest" time="0.034"/>
  <system-out><![CDATA[ClientAccount{id=0, firstName='null', lastName='null', age=0, acId=0, acType='null', acNumber='1010', acClientId=10, acBalance=90000}
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
